react/components/Dialog/DialogMessage.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component, ReactNode } from 'react'
+import { FormattedMessage, InjectedIntlProps, injectIntl, IntlShape } from 'react-intl'
+import { Button } from 'vtex.styleguide'
+import Dialog from './index'
+
+interface DialogMessageProps {
+  message: string
+  onClose: () => void
+  onSuccess: () => void
+  intl?: IntlShape
+}
+
+interface DialogMessageState {
+  isLoading?: boolean
+}
+
+class DialogMessage extends Component<DialogMessageProps & InjectedIntlProps, DialogMessageState> {
+  public state: DialogMessageState = {}
```

Comment:
shouldn't you initialize the `isLoading` here?

Reply:
No, because it starts with false.

Reply:
I meant to initialize it as false here

```tsx
public state: DialogMessageState = {
  isLoading: false,
}
```